docker.go

```diff
@@ -178,6 +190,16 @@ func (p Plugin) Exec() error {
 		cmds = append(cmds, commandPull(img))
 	}
 
+	// setup for using ssh agent (https://docs.docker.com/develop/develop-images/build_enhancements/#using-ssh-to-access-private-data-in-builds)
+	if p.Build.SSHAgent != "" {
+		// TODO check in with one of the drone devs...this should not be necessary. I'm probably doing something
+		// wrong with the cli framework
+		p.Build.SSHAgent = strings.TrimSuffix(p.Build.SSHAgent, "]")
```

Comment:
Not sure what I'm missing here @tphoney -- I'm getting my default value for a flag wrapped in `[]`